data/json/monstergroups/mutant_upgrades.json

```
@@ -48,10 +48,10 @@
   {
     "name": "GROUP_RATKIN_EVOLVED",
     "type": "monstergroup",
-    "default": "mon_black_rat",
+    "default": "mon_black_rat2",
```

This needs a descriptive name, here's a suggestion.
```suggestion
    "default": "mon_black_ratkin",
```